daemon/audit.c

```diff
@@ -176,26 +175,12 @@ audit_send_record_cb(const char *hash_string, const char *hash_file,
 		return;
 	}
 
-	uint32_t buf_len = file_size(hash_file);
-	buf = mem_alloc0(buf_len);
-
-	int read = file_read(hash_file, (char *)buf, buf_len);
-
-	if (read < 0 || buf_len != (unsigned int)read) {
-		ERROR("Processing SCD response: read %u bytes, expected %u", read, buf_len);
-		goto out;
-	}
-
-	TRACE("Got hash from SCD for file %s: %s", hash_file, hash_string);
-
-	if (unlink(hash_file)) {
-		ERROR_ERRNO("Failed to unlink %s", hash_file);
-	}
+	TRACE("Got hash from SCD for buf %s: %s", hash_buf, hash_string);
```

Comment:
imho we should use TRACE_HEXDUMP here to print non-ascii buffers properly

Reply:
makes sense.